pkg/rpm.bzl

```diff
@@ -142,7 +142,8 @@ pkg_rpm = rule(
             mandatory = True,
             allow_single_file = spec_filetype,
         ),
-        "architecture": attr.string(default = "all"),
+        "architecture": attr.string(default = "noarch"),
```

Comment:
In the long run, we should be aiming to help people built the spec file, so we want to be able to create a variable to define *BuildArch*.  Perhaps the default for this could be '@auto@'.  And within the RpmBuilder __init__() we compute arch. If arch is @auto@ we check target and parse out the first word. If no target, then we use 'noarch'